packages/api/src/routes/internal.ts

```diff
@@ -121,39 +119,6 @@ router.post(
   })
 );
 
-/** ---------------------------------------------------------------------------
- * POST /internal/redrive-sidechain-dlq
- *
- * Pull messages from the sidechain DLQ, unique, and reprocess them.
- *
- * @param req.query.maxNumberOfMessages - The maximum number of messages to pull. Defaults to all (-1).
- * @return 200 When successful, including the original and unique counts.
- */
-router.post(
-  "/redrive-sidechain-dlq",
-  asyncHandler(async (req: Request, res: Response) => {
-    const maxNumberOfMessagesRaw = getFrom("query").optional("maxNumberOfMessages", req);
-    const maxNumberOfMessages = maxNumberOfMessagesRaw ? parseInt(maxNumberOfMessagesRaw) : -1;
-
-    const result = await redriveSidechainDLQ(maxNumberOfMessages);
-    return res.json(result);
-  })
-);
-
-/** ---------------------------------------------------------------------------
- * GET /internal/peek-sidechain-dlq
- *
- * Read the first 10 messages from the sidechain DLQ without removing them, and return a link
- * to download the files they point to.
- */
-router.get(
-  "/peek-sidechain-dlq",
```

Comment:
Removing these as we don't use them. The redrive has been replaced by the `sqsmover` CLI utility. The peek needs work but should really be a local script (moved to `packages/utils` in the meantime).